include/glow/Quantization/Base/Base.h

```diff
@@ -99,6 +101,11 @@ QuantizationTransform32To8 quantizeScaleOffset32To8(float scale,
 TensorQuantizationParams chooseQuantizationParams(float min, float max,
                                                   Schema schema = Asymmetric);
 
+/// \return an int8 vector mapping from the \p inTy to the \p outTy given the
```

Comment:
super nit: \returns

Maybe specify in the \pre that inTy and outTy should be int8qty types.

Reply:
Done